.eslintrc

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
Does the project currently pass all these lint rules? I'd rather not change the way the project is linted in this PR.

If we do want to change the way the project is linted, we should do that in a separate PR.